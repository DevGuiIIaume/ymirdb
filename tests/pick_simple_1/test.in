set a -1 0 1
pick a 1
pick a 2
pick a 3
append a 2
pick a 4
push a -2
pick a 1
pick a 2
pick a 3
pick a 4
pick a 5
pick a 6
list entries
bye
