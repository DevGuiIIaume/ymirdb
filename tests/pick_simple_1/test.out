> ok

> -1

> 0

> 1

> ok

> 2

> ok

> -2

> -1

> 0

> 1

> 2

> index out of range

> a [-2 -1 0 1 2]

> bye
